src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -89,6 +91,35 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
         /// </summary>
         public string RuntimeMetadataVersion { get; private set; }
 
+        // 1.2 BACKCOMPAT OVERLOAD -- DO NOT TOUCH
```

Comment:
I followed the guidelines outlined in: https://github.com/dotnet/roslyn/blob/master/docs/Adding%20Optional%20Parameters%20in%20Public%20API.md